index.md

```diff
@@ -1,19 +1,24 @@
 ---
 layout: default
 ---
-# Hackathon Hackers Europe
-We're hosting a casual meet-up for UK and European hackathon organisers!
+# Hackathon Hackers Europe 
+Hackathon Hackers Europe (or HHEU for short) is an organisation targetting at helping student lead hackathons around europe. We are here to bring the hackatoh organiser and attendee community together to make hackathons easier to run and partipate in.
 
-This **unconference** is a chance to meet and chat with other hackathon organisers from across the UK. 
+HHEU put on events throughout the year to help meet our objectives like the <a href="/HHEUConf.html">HHEU conference</a>.
```

Comment:
Jekyll builds redirects such that we don't need the `.html`. This can just link to `/HHEUConf` (although I'd be tempted to make it lowercase, i.e. `hheu-unconf`)